### ISSUE TITLE
Bullets page accessibility

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#922]
For discussion during the next developer day...

Alistair suggested removing the bullets page because it potentially fails WCAG 2.2.2 https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

Obviously removing the bullets page is not viable or desirable because it will have been used in lots of existing learning objects. I don't think we should make it a deprecated page either especially after all the work done on it by @FayCross and @JohnSmith-LT!

I wonder if we could/should add links/buttons/keyboard shortcuts to pause, resume, show all as a solution for this?

**Comment by FayCross** [comment]
I'll look at adding these

**Comment by FayCross** [comment]
For info, this issue is noted on the 3.14 accessibility statement:

The Bullets page in XOT projects gradually adds blocks of information to the screen at regular intervals. The timing of the appearance of these bullets cannot be changed by the end-user. This fails [2.2.2 (Pause, Stop, Hide)](https://www.w3.org/WAI/WCAG22/quickref/#pause-stop-hide). However, as blocks of content are only added and not removed from the screen, users are not meaningfully disadvantaged.
